package_nso_to_oc/xe/xe_system.py

```diff
@@ -862,6 +864,31 @@ def cleanup_server_access(config_leftover, group_access_type, access_type):
     elif "server" in config_leftover.get(f"tailf-ned-cisco-ios:{access_type}", {}):
         del config_leftover[f"tailf-ned-cisco-ios:{access_type}"]["server"]
 
+def xe_system_clock_timezone(config_before: dict, config_leftover: dict) -> None:
+    """
+    Translates NSO XE NED to MDD OpenConfig System Clock Timezone
+    """
+    openconfig_system_clock_config = openconfig_system["openconfig-system:system"][
+        "openconfig-system:clock"]["openconfig-system:config"]
+    timezone_name = ["UTC", "0", "0"] # Placeholder Timezone, Hours and Minutes
+    
+    if config_before.get("tailf-ned-cisco-ios:clock", {}).get("timezone", {}).get("zone") and len(
+        config_before.get("tailf-ned-cisco-ios:clock", {}).get("timezone", {}).get("zone")) == 3:
+        timezone_name[0] = config_before.get("tailf-ned-cisco-ios:clock", {}).get("timezone", {}).get("zone")
+        del config_leftover["tailf-ned-cisco-ios:clock"]["timezone"]["zone"]
+
+    if config_before.get("tailf-ned-cisco-ios:clock", {}).get("timezone", {}).get("hours") or config_before.get(
+        "tailf-ned-cisco-ios:clock", {}).get("timezone", {}).get("hours") == 0:
```

Comment:
The if statement can then be simplified to...

```suggestion
    if hours != None:
```